.github/workflows/merge_to_main_staging.yml

```
@@ -55,6 +55,7 @@ env:
   TF_VAR_client_vpn_access_group_id: ${{ secrets.STAGING_CLIENT_VPN_ACCESS_GROUP_ID }}
   TF_VAR_client_vpn_saml_metadata: ${{ secrets.STAGING_CLIENT_VPN_SAML_METADATA }}
   TF_VAR_client_vpn_self_service_saml_metadata: ${{ secrets.STAGING_CLIENT_VPN_SELF_SERVICE_SAML_METADATA }}
+  TF_VAR_gha_arc_pat: ${{ secrets.GHA_ART_PAT }}
```

Ok is there really a typo and this was copy / pasted, or it's all on purpose?
```suggestion
  TF_VAR_gha_arc_pat: ${{ secrets.GHA_ARC_PAT }}
```

---

Yeah, I digitized @patheard and threw him into a secret. Now he will work forever!